core/src/crds_value.rs

```diff
@@ -235,6 +237,37 @@ impl Version {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct ValidatorGroup {
+    pub from: Pubkey,
+    pub set: Vec<Pubkey>,
```

Comment:
So for gossip messages it's critical that the overall size stays under MTU. 

We ensure this by making sure all possible `CrdsValue` stay under `MAX_PROTOCOL_PAYLOAD_SIZE `: https://github.com/solana-labs/solana/blob/master/core/src/cluster_info.rs#L88. If you're interested in how that's derived, see discussion here: https://discord.com/channels/428295358100013066/478692221441409024/729799349323956244

So here I think we should:

1) Change the vector to a fixed size array
2) Write a test similar to `test_vote_size`: https://github.com/solana-labs/solana/blob/master/core/src/cluster_info.rs#L3353 that asserts the maximal size of such a value stays under the maximal limit.